packages/@aws-cdk/aws-s3/lib/auto-delete-objects-handler/index.ts

```diff
@@ -0,0 +1,43 @@
+export async function handler(event: AWSLambda.CloudFormationCustomResourceEvent) {
+  // eslint-disable-next-line @typescript-eslint/no-require-imports, import/no-extraneous-dependencies
+  const s3 = new (require('aws-sdk').S3)();
+  if (event.RequestType === 'Create') { return onCreate(s3, event); }
+  if (event.RequestType === 'Update') { return onUpdate(s3, event); }
```

Comment:
nitpick: We can avoid defining the empty `onCreate` and `onUpdate` handlers and directly return from here. This will make it clear that only `Delete` case is handled in this CustomResource.

Reply:
Sure, why not.

Reply:
A `switch/case` could also work here:
```ts
  switch (event.RequestType) {
    case 'Create':
    case 'Update':
      return;

    case 'Delete':
      return onDelete(event);
  }
```